src/main/resources/templates/cypher/list.html

```diff
@@ -49,15 +51,20 @@ <h2>Šifry</h2>
                 <td th:text="${cypher.trapCodeword}">Past</td>
                 <td th:text="${cypher.bonusContent}">Bonus</td>
                 <td align="center">
-                    <a href="#" th:href="@{/admin/hint(cypherId=${cypher.cypherId})}">Detail</a> |
-                    <a href="#" th:href="@{/admin/cypher/update/{cypherId}(cypherId=${cypher.cypherId})}">Upravit</a> |
-                    <a href="#" th:href="@{/admin/cypher/delete/{cypherId}(cypherId=${cypher.cypherId})}">Smazat</a>
+                    <button class="btn dropdown-togle" type="button" data-toggle="dropdown">Akce
```

Comment:
Pridat tridu `btn-primary`

Reply:
Typo: dropdown-togle -> dropdown-toggle

Reply:
a ja si rikal, proc to tak blbe vypada, diky :) upraveno